lib/api.ts

```diff
@@ -191,10 +197,27 @@ export function getCalendarData(activity: Activity[]) {
         };
       }
       acc[date].count += 1;
+
+      const uniquePrReviews = new Set();
+
       if (acc[date][activity.type]) {
-        acc[date][activity.type] += 1;
+        if (
+          activity.type != "pr_reviewed" ||
+          (activity.type == "pr_reviewed" &&
+            uniquePrReviews.has(activity.title))
```

Comment:
this has to be 
` !uniquePrReviews.has(activity.title))`
the condition should not be true
but keeping not true leads incorrect not sure why
but removing the not true gives correct pr_reviews values, even i checked manually